Alignment/OfflineValidation/interface/PrepareDMRTrends.h

```diff
@@ -0,0 +1,185 @@
+#ifndef PREPAREDMRTRENDS_H_
```

Comment:
```suggestion
#ifndef ALIGNMENT_OFFLINEVALIDATION_PREPAREDMRTRENDS_H_
```
please use full-fledged inclusion guards to minimize chances of collisions.